tools/docker/Dockerfile

```diff
@@ -168,13 +168,31 @@ COPY tools/docker/naive-snapshotter/entrypoint.sh /entrypoint
 ENTRYPOINT ["/entrypoint"]
 CMD ["exec /bin/bash"]
 
+FROM firecracker-containerd-test as firecracker-containerd-devmapper-integ-test
```

Comment:
I think I'd prefer we take a different approach than adding a new test image for this. We could update to not start or configure containerd as part of the test image at all and instead have each test case configure+start containerd itself (as part of the `prepareTest` util you recently added).

That approach will give each test case a lot more flexibility and keep our test-image infra much simpler (we can continue just using one single test image instead of managing multiples). 

I think there is more effort required upfront in order to do this, but I suspect it's worth it for longer-term maintenance. However, if it turns out for some reason to be an enormous effort way out of the scope of just using the devmapper, then I'm fine with it just being a backlog issue.

Reply:
While I agree that we don't have to have an additional Docker image for running the tests, I'm unsure that starting containerd itself from prepareTest is a good way.

Currently integration tests are act as clients, but starting containerd from there makes the tests as servers.

Let me take a look how hard it would be.

Reply:
I was imagining the tests start containerd by just shelling out, so the tests themselves would not be servers. They would start a server and then use a client to interact with/test that server. But yeah, let's see how much effort it would be.

Reply:
Okay. Renaming the Docker image to be snapshotter-agonostic is in #301.

Reply:
Unified the Docker image